src/components/elements/Checkbox/index.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react'
```

Comment:
well, as I said before, this is preference. I am using this convention in other projects, but not all people do like this. basically it has jsx with js, not js with jsx in my opinion. since exporting data is extended to react component one, so I thought it is more clear to differentiate with jsx extension. Again, this is preference. If you'd like js extension, I definitely can do that. What I usually do, containers and components are jsx, actions and reducers are js.